src/kernel/default.ts

```diff
@@ -610,7 +259,7 @@ class Kernel implements IKernel {
    *
    * If the kernel status is `Dead`, this will throw an error.
```

Comment:
In the doc strings it says `Dead` but in the code it's `dead` ... I don't know if this makes a big difference, but we may want to sync those up.